homeassistant/components/repairs/__init__.py

```diff
@@ -6,16 +6,27 @@
 
```

Comment:
An alternative to this is that we could add something like this to the `RepairsFlow` base class:

```python
    @property
    def flow_manager(self) -> RepairsFlowManager:
        """Return the repairs flow manager"""
        return self.hass.data[DOMAIN]["flow_manager"]
```

That would make it accessible in all flows, but I figured a helper was OK now while the use case is fairly limited.

Reply:
Sounds good to start with the helper and see where it goes.